privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -55,8 +56,9 @@ mobileCloseBentoButtonTitle = Close menu
 ## Home Page - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/
 
 home-hero-headline = Hide your real email address to help protect your identity
-home-hero-copy = { -brand-name-firefox-relay } makes it easy to create aliases, randomly generated email addresses that forward to your real inbox. Use it to protect your online accounts - and your identity - from hackers. Sign in with your { -brand-name-firefox-account } to get started.
-home-hero-cta = { nav-profile-sign-in }
+home-hero-copy = { -brand-name-firefox-relay } makes it easy to create aliases, randomly generated email addresses that forward to your real inbox.
+    Use it to protect your online accounts - and your identity - from hackers. Sign in with your { -brand-name-firefox-account } to get started.
```

Comment:
This needs to have a string ID, maybe make the string IDs as 
- `home-hero-copy-1 =`
- `home-hero-copy-2 =`

Can you split it up like this? I think it is ok to combine this one. The ones with line breaks can be broken up.